src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -79,6 +79,11 @@ const StyledButton = styled(Button)`
     margin-top: 8px;
     max-width: 120px;
 `
+const StyledRefLink = styled(Flex)`
+    @media ${device.mobileL} {
+        justify-content: left;
+    }
+`
```

Comment:
You can use the `mobileL` prop for this.

Reply:
Im sticking to this comp as I tried to use `mobileL` prop but its not working.

Reply:
Ah, seems like the flex properties haven't been added to the generate responsive styles function in `Box`. Sure, we will pick that one up next. 👌